### PR TITLE
4966250: SSLSessionContext.setSessionTimeout() documentation could be updated

### PR DESCRIPTION
**Pull request by mcpowers**
https://bugs.openjdk.org/browse/JDK-4966250

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change requires CSR request [JDK-8333311](https://bugs.openjdk.org/browse/JDK-8333311) to be approved

### Issues
 * [JDK-4966250](https://bugs.openjdk.org/browse/JDK-4966250): SSLSessionContext.setSessionTimeout() documentation could be updated (**Bug** - P4)
 * [JDK-8333311](https://bugs.openjdk.org/browse/JDK-8333311): SSLSessionContext.setSessionTimeout() documentation could be updated (**CSR**)

### Reviewers
 * [Sean Mullan](https://openjdk.org/census#mullan) (@seanjmullan - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/19458/head:pull/19458` \
`$ git checkout pull/19458`

Update a local copy of the PR: \
`$ git checkout pull/19458` \
`$ git pull https://git.openjdk.org/jdk.git pull/19458/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 19458`

View PR using the GUI difftool: \
`$ git pr show -t 19458`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/19458.diff">https://git.openjdk.org/jdk/pull/19458.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/19458#issuecomment-2137741863)

### CODE REVIEW

**Comment by bradfordwetmore** [comment]
This needs a CSR before pushing.

File: src/java.base/share/classes/javax/net/ssl/SSLSessionContext.java [hunk 1]
```diff
@@ -88,8 +88,9 @@ public interface SSLSessionContext {
      * If the timeout limit is set to 't' seconds, a session exceeds the
      * timeout limit 't' seconds after its creation time.
      * When the timeout limit is exceeded for a session, the
-     * <code>SSLSession</code> object is invalidated and future connections
-     * cannot resume or rejoin the session.
+     * <code>SSLSession</code> object is marked so that future connections
+     * cannot resume or rejoin the session. Sessions can continue to be used
```
Comment (seanjmullan) [inline src/java.base/share/classes/javax/net/ssl/SSLSessionContext.java:92]: Maybe make it more clear these are active sessions -  "Active sessions can continue to ..."

Comment (mcpowers) [reply src/java.base/share/classes/javax/net/ssl/SSLSessionContext.java]: Good idea - changed.

**Review by seanjmullan — APPROVED** [review]
Looks good. Please add a noreg-doc label to the bug and write a release note.

**Pull request closed without merging**